### DISCUSSION TITLE
Virtual barcode support

### DISCUSSION THREAD

**Discussion by vranki** [TampereHacklab/mulysa#380] [Ideas]
Generate virtual barcodes that can be copypasted to banking software.

**Comment by vranki** [comment]
https://www.finanssiala.fi/maksujenvalitys/dokumentit/Pankkiviivakoodi-opas.pdf

**Comment by olmari** [comment]
Kitupiikin toteutustapa, kaavaa voinee noudattaa Mulysassakin: https://github.com/artoh/kitupiikki/blob/master/kitupiikki/laskutus/laskuntulostaja.cpp#L239-L258
